src/features/quiz/ui/Question.tsx

```diff
@@ -1,22 +1,25 @@
-import { QuestionDiv, QuestionSection, TextBlockButton } from './../styles';
+import { QuestionSection, TextBlockButton, Titlediv } from './../styles';
 import { useClientQuizStore } from '../../../store/useClientQuizStore';
 import parse, { HTMLReactParserOptions, Element } from 'html-react-parser';
 import '../styles.css';
 import emptyChangeToDiv from '../service/emptyChangeToDiv';
 import lineChanger from '../service/lineChanger';
 import { useRef } from 'react';
 import Dompurify from 'dompurify';
-interface questiontype {
-  title: string;
-  question: string;
-  category: string;
+import Quiz from '../../../types/Quiz';
+interface questionProps {
```

Comment:
```suggestion
interface QuestionProps {
```
단순 오타겠죠 ㅎㅎㅎ ? 인터페이스 네이밍은 파스칼 케이스로 작성되는게 일반적입니다 합니다

Reply:
아이고.. 오타인것 같습니다
꼼꼼하게 봐주셔서 감사합니다!